src/modules/RA/BaseLayer/components/ZoomInput/ZoomInput.js

```diff
@@ -0,0 +1,41 @@
+import React, { useCallback } from 'react';
+import { useField, useForm } from 'react-final-form';
+
+import RangeInput from '../../../../../components/react-admin/RangeInput';
+
+const ZoomInput = ({
+  basePath,
+  record,
+  resource,
+  initialValue: [valueMin, valueMax],
```

Comment:
Can you add field path properties with default to current value PLS ?

Reply:
I don't understand the request

Reply:
I remove `basePath`, `record`, and `ressource` from props to spread the rest to `RangeInput`

Reply:
That's not what I mean ;-) Can you allow the field name of min zoom and max zoom

Reply:
Ok fixed